login/index.php

```diff
@@ -185,15 +186,23 @@
 						</div>
 
 					</div>
-					<div id="password" class="input-group mb-3">
-						<div class="input-group-prepend">
-							<span class="input-group-text" id="basic-addon1"><i class="p-1 fa-solid fa-key login-icon"></i></span>
-						</div><input type="password" id="passinp" class="form-control login-input" name="password" placeholder="Password">
-						<div style="font-size:20px" id="pass_inv" class="invalid-feedback">
-						Password cannot be empty.
-						</div>
-
-					</div>
+<div id="password" class="input-group mb-3">
+    <div class="input-group-prepend">
+        <span class="input-group-text" id="basic-addon1">
+            <i class="p-1 fa-solid fa-key login-icon"></i>
+        </span>
+    </div>
+    <input type="password" id="passinp" class="form-control login-input" name="password" placeholder="Password">
+    
+
+    <button class="btn btn-outline-secondary" type="button" id="togglePass" onclick="togglePassword()">
+        <i class="fa-solid fa-eye" id="toggleIcon"></i>
+    </button>
```

Comment:
This should align with the indentation of the existing HTML
```suggestion

					<div id="password" class="input-group mb-3">
						<div class="input-group-prepend">
							<span class="input-group-text" id="basic-addon1">
								<i class="p-1 fa-solid fa-key login-icon"></i>
							</span>
						</div>
						<input type="password" id="passinp" class="form-control login-input" name="password" placeholder="Password">
							<button class="btn btn-outline-secondary" type="button" id="togglePass" onclick="togglePassword()">
								<i class="fa-solid fa-eye" id="toggleIcon"></i>
							</button>
```